pkg/arvo/sys/vane/gall.hoon

```diff
@@ -42,9 +42,33 @@
 ::  $move: Arvo-level move
 ::
 +$  move  [=duct move=(wind note-arvo gift-arvo)]
+::
++$  userspace-move  [=duct move=(wind userspace-note:agent userspace-gift:agent)]
```

Comment:
what do you think of renaming all the `*-userspace` types to `*-user` instead?

Reply:
(and `userspace-*` to `user-*`)

Reply:
Sounds good to me, done